components/cluster/libstatgrab/Makefile

```diff
@@ -36,11 +36,11 @@ PATCH_LEVEL= 1
 COMPONENT_PRE_CONFIGURE_ACTION=($(CLONEY) $(SOURCE_DIR) $(@D))
 
```

Comment:
I'll stop here with my COMPONENT_REVISION comments.
The general rule is: If you change anything either set the new COMPONENT_VERSION (and delete COMPONENT_REVISION) or set/increment COMPONENT_REVISION (starting at 1).